core/utils.py

```diff
@@ -1198,8 +1207,8 @@ def compute_list_difference(list_a: List[str], list_b: List[str]) -> List[str]:
     return list(sorted(set(list_a) - set(list_b)))
 
 
-# Ignoring type-arg because error thrown is 'Missing type parameters for generic
-# type "OrderedDict"' but here we don't need to specify this.
+# Here we use MyPy ignore because error thrown is 'Missing type parameters
```

Comment:
I don't understand the rationale, can you elaborate more on this?

Reply:
OK, I think you can still explain in the code comment why "we don't need to specify this".

Reply:
Comment Updated!